frontend/src/wizards/Migration/sections/ComputeCompatSection.tsx

```diff
@@ -0,0 +1,50 @@
+/* Copyright Contributors to the Open Cluster Management project */
+import React from 'react'
+import { FormGroup, Title } from '@patternfly/react-core'
+
+interface Props {
+  srcCompute: string
+  dstCompute: string
+}
+
+export const ComputeCompatSection: React.FC<Props> = ({ srcCompute, dstCompute }) => (
```

Comment:
wouldn't you follow the `export function ComputeCompatSection(props: Props)` approach instead? I would also rename the `Props` interface by `ComputeCompatSectionProps`
wdyt?

Reply:
the same for the rest of the components